### DISCUSSION TITLE
[Feature Request]: 电视剧-热门订阅将动漫和剧集进行分类

### DISCUSSION THREAD

**Discussion by alekzander98** [jxxghp/MoviePilot#3351] [讨论]
### 当前程序版本

v1.9.2

### 运行环境

Docker

### 功能改进类型

主程序

### 功能改进

是否可以考虑电视剧热门订阅将动漫跟剧集分开，比如说增加一个热门动漫tab，感觉现在看着有点乱。

**Comment by cng10001** [comment]
em~  被二次元占领嘞。好像j大不看动漫，但是mp用户二次元用的很多。

现在首页14部剧集，电视剧和动漫的比例是4：10

支持给二刺螈开个专版！

![电视剧分类 拷贝](https://github.com/jxxghp/MoviePilot/assets/115156545/4ba524a3-aa5f-4be2-a55b-029d632f273c)

**Comment by Travel2Here** [comment]
复议。纯粹是因为混乱~

**Comment by zoffyultraman** [comment]
看起来和热度有关，但确实支持开个专区

**Comment by jxxghp** [comment]
没想到动漫群体这么强大，尴尬的是之前数据汇报没有区分动漫类型，已存储的数据上区分不出来了（一个个去查一遍也不太可行）。

**Comment by alekzander98** [comment]
> 没想到动漫群体这么强大，尴尬的是之前数据汇报没有区分动漫类型，已存储的数据上区分不出来了（一个个去查一遍也不太可行）。

<img width="808" alt="image" src="https://github.com/jxxghp/MoviePilot/assets/35864786/e875e5c4-9f0c-45ce-a6c6-651eba65f482">
看到对象中有返回对应TMDB的媒体ID，只是genre_ids现在为空，是否可以考虑跑批补全genre_ids字段，或者设置一个定时任务去更新呢
